client/src/components/Heading/index.tsx

```diff
@@ -0,0 +1,15 @@
+import styles from './style.module.scss';
+import { PropsWithChildren } from 'react';
+
+interface HeadingProps {
+  centered?: boolean;
+}
+
+/**
+ * Unlike `Typography`, the text size for the heading shrinks on mobile.
+ */
+const Heading = ({ centered, children }: PropsWithChildren<HeadingProps>) => {
+  return <h1 className={`${styles.heading} ${centered ? styles.centered : ''}`}>{children}</h1>;
+};
```

Comment:
Hm, I wonder if we can also just have shrinking for Typography as well, I remember doing this for some other repo

https://github.com/acmucsd/outreach-website/tree/main/src/components/Typography

 I think this is more of a consideration when we start making a component library so we can have this be standardized